### ISSUE TITLE
Smart Replies

### ISSUE THREAD

**Issue by garysahota** [klinker-apps/messenger-issues#856]
**Describe your request:**

I would like to add smart replies to our texting options via the (newly released) ML Kit:

https://9to5google.com/2019/04/05/ml-kit-smart-reply/
https://github.com/jeziellago/smart-reply-mlkit

**Comment by klinker24** [comment]
Yes, I saw that this was released. I plan to include them in the conversation UI, and give the user the option to expose them through notification actions, as well. 

Todo:
* [x] refactor notifications to give the user more fine-grained control over what actions appear (the user should be able to set up actions for: reply, smart reply, smart reply. 
* [x] Integrate smart replies into the notifications
* [ ] Make preference in the "Advanced Features" section to allow the user the choice of whether or not they want to use these smart replies
* [ ] Generate and integrate the smart replies into the conversation list UI
* [ ] Make help topics around the notification actions refactor, using the smart replies through the notification actions, and using the smart replies within the app

At this time, smart replies are not available on any platforms, other than Android. Unless Google makes a JavaScript version, it will remain this way.
